sdformat_urdf/test/geometry_tests.cpp

```diff
@@ -0,0 +1,256 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#include <gtest/gtest.h>
+#include <sdf/sdf.hh>
+#include <sdformat_urdf/sdformat_urdf.hpp>
+#include <urdf_model/model.h>
+#include <urdf_model/types.h>
+
+#include "sdf_paths.hpp"
+#include "test_tools.hpp"
+
+TEST(Geometry, geometry_box)
+{
+  sdf::Errors errors;
+  urdf::ModelInterfaceSharedPtr model = sdformat_urdf::parse(
+    get_file(PATH_TO_SDF_GEOMETRY_BOX), errors);
+  EXPECT_TRUE(errors.empty()) << errors;
+  ASSERT_TRUE(model);
+  ASSERT_EQ("geometry_box", model->getName());
+
+  urdf::LinkConstSharedPtr link = model->getRoot();
+  ASSERT_NE(nullptr, link);
+  ASSERT_NE(nullptr, link->visual);
+  ASSERT_NE(nullptr, link->collision);
```

Comment:
Could you add an assert for `link->visual->geometry` and `link->collision->geometry` as well? Same for the other tests.

Reply:
Added checks for `->geometry` not being null before using it in 9ae4804